Jenkinsfile

```diff
@@ -371,6 +373,22 @@ def buildMacOS(workerFunction) {
   }
 }
 
+def buildMacOSArm(workerFunction) {
+  return {
+    myNode('osx_vegas') {
+      withEnv([
+        "DEVELOPER_DIR=/Applications/Xcode-12.2.app/Contents/Developer",
+        "NODE_ARCH_ARM=1",
```

Comment:
Do none of the CMake arguments passed by `cmake-js` with `--arch=arm` work here? I don't think a custom environment variable is the best way forward here, if we can somehow hack it to work with just the `--arch` parameter.

Reply:
For non-Windows platforms, cmake-js will ignore `--arch`